AutomatedTesting/Gem/PythonTests/scripting/TestGetSetSCComponent.py

```diff
@@ -0,0 +1,36 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+import azlmbr.math as math
+import editor_python_test_tools.pyside_utils as pyside_utils
+import editor_python_test_tools.hydra_editor_utils as hydra
+import azlmbr.paths as paths
+
+
+class TestGetSetSCComponent:
```

Comment:
Can we get this test class and file name to follow closer naming conventions to the rest of the test suite?